go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubeedge/edgemesh
 
-go 1.16
+go 1.19
```

Comment:
Why we need upgrade go version?

Reply:
There were some dependencies after `go mod tidy` that required at least golang 1.18.

```
make                                            
hack/verify-golang.sh
go detail version: go version go1.16 linux/amd64
go version: 1.16
EDGEMESH_OUTPUT_SUBPATH=_output/local hack/make-rules/build.sh 
building github.com/kubeedge/edgemesh/cmd/edgemesh-agent
+ go build -o /home/armin/edgefarm/kubeedge-edgemesh/_output/local/bin/edgemesh-agent -gcflags= -ldflags '-s -w -buildid= -X github.com/kubeedge/kubeedge/pkg/version.buildDate=2022-10-13T08:12:00Z -X github.com/kubeedge/kubeedge/pkg/version.gitCommit=fef48d8ecf57b88fb5ab65a679a450bedee0a893 -X github.com/kubeedge/kubeedge/pkg/version.gitTreeState=dirty -X github.com/kubeedge/kubeedge/pkg/version.gitVersion=v1.12.0-3+fef48d8ecf57b8-dirty -X github.com/kubeedge/kubeedge/pkg/version.gitMajor=1 -X github.com/kubeedge/kubeedge/pkg/version.gitMinor=12+' github.com/kubeedge/edgemesh/cmd/edgemesh-agent
# go.uber.org/atomic
vendor/go.uber.org/atomic/pointer_go119.go:29:14: type parameter requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:29:16: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:31:18: type instantiation requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:35:17: type parameter requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:35:19: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:35:38: type instantiation requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:44:17: type instantiation requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:49:17: type instantiation requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:54:17: type instantiation requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:59:17: type instantiation requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/go.uber.org/atomic/pointer_go119.go:59:17: too many errors
# github.com/marten-seemann/qtls-go1-19
vendor/github.com/marten-seemann/qtls-go1-19/cipher_suites.go:141:43: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/cipher_suites.go:398:45: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/cipher_suites.go:403:46: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/cipher_suites.go:411:45: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/conn.go:178:10: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/conn.go:184:13: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/conn.go:212:62: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/conn.go:976:14: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/conn.go:1059:33: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/common.go:1501:41: predeclared any requires go1.18 or later (-lang was set to go1.16; check go.mod)
vendor/github.com/marten-seemann/qtls-go1-19/conn.go:1059:33: too many errors
make: *** [Makefile:61: all] Error 2
```

Reply:
rewind back to go 1.16 again